siem_integrations/README.md

```diff
@@ -26,10 +26,103 @@ In order to utilize splunk2kafka, a [running Kafka instance](https://kafka.apach
 2. Install export2kafka ([Instructions](https://github.com/rapidsai/clx-siem-integration/blob/master/splunk2kafka/export2kafka/README.md))
 
 
-## CLX Query
-
-Coming soon...
-
+## CLX Query Sample Dataset
+
+1. Download sample dataset.
+    
+    ```aidl
+    wget http://files.grouplens.org/datasets/movielens/ml-25m.zip
+    ```
+   
```

Comment:
Is there more that will be here? Otherwise, I'd remove the number and just include (1) a description of the dataset and (2) a link to download it.

Reply:
Updated as suggested. Thank you!